zhaquirks/philips/soc001.py

```diff
@@ -0,0 +1,45 @@
+"""Signify SOC001 device."""
+
+from zigpy import types
+from zigpy.quirks import CustomCluster
+from zigpy.quirks.v2 import BinarySensorDeviceClass, ClusterType, QuirkBuilder
+
+
+class PhilipsContactCluster(CustomCluster):
+    """Philips manufacturer specific cluster for contact sensor."""
+
+    cluster_id = 64518  # 0xfc06
+    name = "Philips contact cluster"
+    ep_attribute = "philips_contact_cluster"
+    attributes = {
+        0x0100: ("contact", types.enum8, True),
+        0x0101: ("last_contact_change", types.uint32_t, True),
+        0x0102: ("tamper", types.enum8, True),
+        0x0103: ("last_tamper_change", types.uint32_t, True),
+    }
+
+
+(
+    #  <SimpleDescriptor endpoint=2 profile=260 device_type=1026
+    #  device_version=0
+    #  input_clusters=[0, 1, 3, 64518]
+    #  output_clusters=[0, 3, 6, 25]>
+    QuirkBuilder("Signify Netherlands B.V.", "SOC001")
+    .removes(cluster_id=6, endpoint_id=2, cluster_type=ClusterType.Client)
+    .replaces(
+        replacement_cluster_class=PhilipsContactCluster, cluster_id=64518, endpoint_id=2
```

Comment:
We do not need to specify `cluster_id` here, as it's the same as in the custom `PhilipsContactCluster`.
(It removes the cluster with the `cluster_id` specified here and then adds the new custom cluster with the `cluster_id` specified in it. Since it's the same for both, we can remove the `cluster_id` here.)

We can probably also drop the first argument being a keyword one. So something like this should be enough:
```python
    .replaces(PhilipsContactCluster, endpoint_id=2)
```

Reply:
Changed in : https://github.com/zigpy/zha-device-handlers/commit/5cd4857d73077c6bb0cb4ac1009533690db10919